docs/int/quickstart/group/quickstart-group-operator.md

```diff
@@ -189,6 +189,41 @@ Exiting your validator(s) can be useful in situations where you want to stop sta
 
 👉 Follow the exit guide [here](docs/int/quickstart/quickstart-exit.md)
 
+## Updating DVT stack
+
+It is highly recommended to upgrade your DVT stack from time to time. To do this, follow these steps:
+```
+# Change to the node directory
+cd charon-distributed-validator-node
+
+# Pull latest changes to the repo
+git pull
+
+# Restart your DVT stack!
+docker compose restart -d
+```
+
+You may get a `git conflict` error like this:
```

Comment:
great that you included this, I think we can make it a bit more general by saying any custom changes will create issues and how to resolve them such as for example prom token but could also be custom BN, custom sync point, etc